### DISCUSSION TITLE
Layerno = 1 for the text on the victory screen

### DISCUSSION THREAD

**Discussion by MUGEN-Slayer** [ikemen-engine/Ikemen-GO#1392] [Feature Request]
### Is your feature request related to a problem? Please describe.

With the current layering system, we cannot place layers between portraits and victory text. The only way to do this is to customize the portrait with the layer to create such an illusion:
![zangetsu](https://github.com/ikemen-engine/Ikemen-GO/assets/68881657/e9143e8f-c3f3-4f94-886e-851e71cc85b4)

### Describe the solution you'd like

p1.name.layerno = 1
winquote.layerno = 1

And layerno = 2 for [VictoryBG] elements, if I'm not mistaken the max layerno is "1" and some screenpacks have transition anims. And with the text with higher priority, such anims would need to "level up" as well.

### Search terms

Victory Screen, text

**Comment by Paul45mec** [comment]
I have the same problem, all the time i need to create a portrait with the winquote bg embed on char, but it's very boring.

i think it could be solved with a

Winquote.bg.spr = ???,???

**Reply by MUGEN-Slayer** [reply]
A dedicated bg is not optimal because it is... just one. If priority is applied to the text, then the number of layers between the portrait and the text would have no limit.
